samcli/local/lambda_service/lambda_error_responses.py

```diff
@@ -0,0 +1,89 @@
+""" Common Lambda Error Response"""
+
+import json
+
+from flask import Response
+
+
+class LambdaErrorResponses(object):
+
+    # The content type of the Invoke request body is not JSON.
+    UnsupportedMediaTypeException = ('UnsupportedMediaType', 415)
+
+    # The AWS Lambda service encountered an internal error.
+    ServiceException = ('Service', 500)
+
+    # The resource (for example, a Lambda function or access policy statement) specified in the request does not exist.
+    ResourceNotFoundException = ('ResourceNotFound', 404)
+
+    # The request body could not be parsed as JSON.
+    InvalidRequestContentException = ('InvalidRequestContent', 400)
+
+    @staticmethod
+    def resource_not_found(function_name):
+        """
```

Comment:
I assume you will fill in the doc strings later

Reply:
Yup. Didn't get that far in life yet :)